MainClass.cs

```diff
@@ -0,0 +1,149 @@
+﻿
+
+
+using System;
+
+namespace exc3
+{
+    class MainClass
+    {
+        static List<Item> GetSortedByDate(Refrigerator refrigerator)
+        {
+            List<Item> allItems = new List<Item>();
+            foreach (Shelf shelf in refrigerator.shelves)
+            {
+                allItems.AddRange(shelf.items);
+            }
+
+            return allItems.OrderBy(item => item.expirationDate).ToList();
+        }
+
+        static List<Shelf> SortShelfBySpace(List<Shelf> shelves)
+        {
+            List<Shelf> sortedShelves = shelves.OrderByDescending(shelf => shelf.availableSpace).ToList();
+            return sortedShelves;
+
+        }
+        static List<Refrigerator> SortFridgeBySpace(List<Refrigerator> fridges)
+        {
+            List<Refrigerator> sortedFridges = fridges.OrderByDescending(fridge => fridge.GetAvailableSpace()).ToList();
+            return sortedFridges;
+
+        }
+
+
+
+        public static int Main()
+        {
+
+            Program program = new Program();
+            List<Refrigerator> refrigerators = program.createInfrastructure();
+            Console.Clear();
+            Refrigerator chosenFridge;
+
+            try
+            {
+
```

Comment:
instead of switch, it's better to use Dictionary of num to function. or Better: define an option class with id, description (for manual) and function to execute